docs/ref/models/fields.txt

```diff
@@ -135,25 +152,22 @@ will work anywhere that the ``Student`` model has been imported).
 You can also collect your available choices into named groups that can
 be used for organizational purposes::
 
-    MEDIA_CHOICES = [
-        ('Audio', (
-                ('vinyl', 'Vinyl'),
-                ('cd', 'CD'),
-            )
-        ),
-        ('Video', (
-                ('vhs', 'VHS Tape'),
-                ('dvd', 'DVD'),
-            )
-        ),
-        ('unknown', 'Unknown'),
-    ]
-
-The first element in each tuple is the name to apply to the group. The
-second element is an iterable of 2-tuples, with each 2-tuple containing
-a value and a human-readable name for an option. Grouped options may be
-combined with ungrouped options within a single list (such as the
-`unknown` option in this example).
+    MEDIA_CHOICES = {
```

Comment:
I think we also need to keep the older example around here. ... Lots of historical examples. 

List of tuples mapping a name to a dict of choices would work too I think — that's probably not sensible but if we can find the right phrasing to communicate that it would be good.

Reply:
Good shout - I've added the old example back. While 

```
CHOICES = [
    ("key", {
        "thing": "display
    })
]
```

will work, I think documenting it will become confusing. I personally cannot think of a general way to express that we support:

```
{"Audio": (("vinyl", "Vinyl"),)}
{"Audio": {"vinyl": "Vinyl"}}
("Audio", (("vinyl", "Vinyl"),))
("Audio",  {"vinyl": "Vinyl"})
```

in a way that's not confusing.